tests/unit/modules/test_btrfs.py

```diff
@@ -362,3 +362,300 @@ def test_properties_error(self):
         '''
         self.assertRaises(CommandExecutionError, btrfs.properties,
                           '/dev/sda1', 'subvol', True)
+
+    def test_subvolume_create(self):
+        '''
+        Test btrfs subvolume create
+        '''
+        # Fails when qgroupids is not a list
+        self.assertRaises(CommandExecutionError, btrfs.subvolume_create,
+                          'var', qgroupids='1')
+
+        salt_mock = {
+            'cmd.run_all': MagicMock(return_value={'recode': 0}),
+        }
+        with patch.dict(btrfs.__salt__, salt_mock):
+            self.assertTrue(btrfs.subvolume_create('var', dest='/mnt'))
+            salt_mock['cmd.run_all'].assert_called_once()
+            salt_mock['cmd.run_all'].assert_called_with(
+                ['btrfs', 'subvolume', 'create', '/mnt/var'])
+
+    def test_subvolume_delete_fails_parameters(self):
+        '''
+        Test btrfs subvolume delete
+        '''
+        # We need to provide name or names
+        self.assertRaises(CommandExecutionError, btrfs.subvolume_delete)
+        self.assertRaises(CommandExecutionError, btrfs.subvolume_delete,
+                          names='var')
+
+        # Parameter commit can be 'after' or 'each'
+        self.assertRaises(CommandExecutionError, btrfs.subvolume_delete,
+                          name='var', commit='maybe')
```

Comment:
Please break them into three UT

Reply:
I moved into two, as the two first are testing basically the same.